docs/tutorials/11_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1374 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "43ace962",
+   "metadata": {},
+   "source": [
+    "## Step 1: Importing the packages that we need to tackle the problem"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "dd31bf33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from IPython.display import display, clear_output\n",
+    "from qiskit import Aer, execute, IBMQ\n",
+    "from qiskit.circuit import QuantumCircuit, Parameter\n",
+    "from qiskit.circuit.library import TwoLocal\n",
+    "from qiskit.utils import QuantumInstance, algorithm_globals\n",
+    "from qiskit.opflow import I, X, Y, Z, StateFn, CircuitSampler, PauliExpectation\n",
+    "from qiskit.algorithms import VQE\n",
+    "from qiskit.algorithms.optimizers import COBYLA, SLSQP\n",
+    "from qiskit_nature.operators.second_quantization import FermionicOp\n",
+    "from qiskit_nature.mappers.second_quantization import JordanWignerMapper\n",
+    "from qiskit_nature.converters.second_quantization import QubitConverter\n",
+    "from qiskit_nature.runtime import VQEClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7fff1d3a",
+   "metadata": {},
+   "source": [
+    "## Step 2: Defining some functions that we will use"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc54fdc6",
+   "metadata": {},
+   "source": [
+    "Before we start the definition of the Deuteron Hamiltonian, we need to define a function that performs the behavior of a mathematical function, the Kronecker delta function, which has the following behavior:\n",
+    "\n",
+    "$\\delta_{nm} = \\bigg\\{\\begin{array}{c}0, \\ \\textrm{if} \\ n \\neq m \\\\1, \\ \\textrm{if } \\ n = m.\\end{array}$.\n",
+    "\n",
+    "This function appears in the Deuteron Hamiltonian definition given in [1]. Below we have a piece of code that defines the Kronecker delta function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "af3784b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def kronecker_delta_function(n: int, m: int) -> int:\n",
+    "    \"\"\"A function that performs the behavior of a Kronecker delta function.\n",
+    "\n",
+    "    Args:\n",
+    "        n (int): The first integer index of Kronecker delta function.\n",
+    "        m (int): The second integer index of Kronecker delta function.\n",
+    "\n",
+    "    Returns:\n",
+    "        Returns 1 if n = m, else returns 0.\n",
+    "    \"\"\"\n",
+    "    if type(n) == int and type(m) == int:\n",
+    "        if n == m:\n",
+    "            return 1\n",
+    "        else:\n",
+    "            return 0\n",
```

Comment:
Do you need this output to be an integer? How about simply:
```suggestion
    "        return int(n == m)\n",
```

Reply:
Much better!